tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Logging/Log4Net/DirectSubmission/AppenderCollectionIntegration.cs

```diff
@@ -22,7 +22,7 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Logging.Log4Net.DirectSu
         ReturnTypeName = "log4net.Appender.IAppender[]",
         ParameterTypeNames = new string[0],
         MinimumVersion = "2.0.0",
-        MaximumVersion = "2.*.*",
+        MaximumVersion = "3.*.*",
```

Comment:
I am planning on doing the due diligence to verify that this is correct still.

Reply:
Appears to be good still - not seeing any changes:

https://github.com/apache/logging-log4net/blob/8bf4313cdc7dc2bcc14583c8b206c5a25b7edf4a/src/log4net/Appender/AppenderCollection.cs#L473-L485